### PR TITLE
Update AWS CodeCommit Web URLs

### PR DESCRIPTION
**Pull request by svrx**
The web URLs for AWS CodeCommit used aren't right, they may have changed the pattern. 
Unfortunately there is no configuration available on the repository config to tweak the web URL patterns, which would be good idea anyway at some time in the future. 

For time being I've updated the web URL pattern so it corresponds to URLs used by AWS CodeCommit as of today.

### CODE REVIEW

File: cmd/repo-updater/repoupdater/server_test.go [hunk 1]
```diff
@@ -901,10 +901,10 @@ func TestRepoLookup(t *testing.T) {
 				Description: "The stripe-go lib",
 				VCS:         protocol.VCSInfo{URL: "git@git-codecommit.us-west-1.amazonaws.com/v1/repos/stripe-go"},
 				Links: &protocol.RepoLinks{
-					Root:   "https://us-west-1.console.aws.amazon.com/codecommit/home#/repository/stripe-go",
-					Tree:   "https://us-west-1.console.aws.amazon.com/codecommit/home#/repository/stripe-go/browse/{rev}/--/{path}",
-					Blob:   "https://us-west-1.console.aws.amazon.com/codecommit/home#/repository/stripe-go/browse/{rev}/--/{path}",
-					Commit: "https://us-west-1.console.aws.amazon.com/codecommit/home#/repository/stripe-go/commit/{commit}",
+					Root:   "https://us-west-1.console.aws.amazon.com/codesuite/codecommit/repositories/stripe-go",
```
Comment (mrnugget) [inline cmd/repo-updater/repoupdater/server_test.go:904]:
```suggestion
					Root:   "https://us-west-1.console.aws.amazon.com/codesuite/codecommit/repositories/stripe-go/browse",
```

File: cmd/repo-updater/repoupdater/server.go [hunk 2]
```diff
@@ -699,7 +699,7 @@ func newRepoInfo(r *repos.Repo) (*protocol.RepoInfo, error) {
 			break
 		}
 		region := splittedARN[0]
-		webURL := fmt.Sprintf("https://%s.console.aws.amazon.com/codecommit/home#/repository/%s", region, repo.Name)
+		webURL := fmt.Sprintf("https://%s.console.aws.amazon.com/codesuite/codecommit/repositories/%s", region, repo.Name)
 		info.Links = &protocol.RepoLinks{
 			Root:   webURL,
```
Comment (mrnugget) [inline cmd/repo-updater/repoupdater/server.go:704]:
```suggestion
			Root:   webURL + "/browse",
```

Comment (svrx) [reply cmd/repo-updater/repoupdater/server.go]: Yes, you're right.

**Comment by unknwon** [comment]
@svrx Please follow this comment to sign our CLA since this is the first time you contribute to Sourcegraph: https://github.com/sourcegraph/sourcegraph/pull/10779#issuecomment-630238662

Meanwhile, I'll kick off the CI.

**Comment by svrx** [comment]
> @svrx Please follow this comment to sign our CLA since this is the first time you contribute to Sourcegraph: [#10779 (comment)](https://github.com/sourcegraph/sourcegraph/pull/10779#issuecomment-630238662)
> 
> Meanwhile, I'll kick off the CI.

Is that the link you intended to post?

**Comment by svrx** [comment]
CLA signed, managed to find the link in another issue.

**Comment by mrnugget** [comment]
Thanks @svrx! Sorry about the missing link. Just confirmed that the CLA is signed.

**Pull request merged**